### PR TITLE
code refactor for 16stories4collection and 7stories

### PR DESCRIPTION
**Pull request by supreeth-spiger**
code refactor for 16stories4collection and 7stories

### CODE REVIEW

File: assets/stylesheets/generics/typography.scss [hunk 1]
```diff
@@ -33,7 +33,7 @@ html {
 body {
   font-family: var(--sansTypeface);
   font-weight: var(--regular);
-  font-size: var(--base);
+  // font-size: var(--base);
```
Comment (nandap4790) [inline assets/stylesheets/generics/typography.scss]: remove this

File: src/components/atoms/BigStoryCard/bigStoryCard.m.css [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
   display: grid;
-
-  grid-template-columns: 1fr 1fr;
-
+  grid-template-rows: 1fr;
   @media (min-width: 768px) {
     grid-template-columns: 1fr;
   }
+  :global .headline {
+    /* @include font-styles(h5); */
```
Comment (nandap4790) [inline src/components/atoms/BigStoryCard/bigStoryCard.m.css]: remove this

File: src/components/atoms/BigStoryCard/bigStoryCard.m.css [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
+    margin: 0px;
```
Comment (nandap4790) [inline src/components/atoms/BigStoryCard/bigStoryCard.m.css]: margin: 0;

File: src/components/atoms/ReadMoreLink/readMoreLink.m.css [hunk 4]
```diff
@@ -1,31 +1,18 @@
...
-      transform: translateX(var(--space-4));
-    }
-  }
+  font-size: var(--space2);
+  font-weight: normal;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 2.5;
+  letter-spacing: normal;
+  color: #333333;
```
Comment (nandap4790) [inline src/components/atoms/ReadMoreLink/readMoreLink.m.css]: get the colors

File: src/components/atoms/ReadMoreLink/readMoreLink.m.css [hunk 5]
```diff
@@ -1,31 +1,18 @@
...
+  height: 40px;
+  border-radius: 3px;
+  border: solid 1px #797979;
```
Comment (nandap4790) [inline src/components/atoms/ReadMoreLink/readMoreLink.m.css]: colors

File: src/components/atoms/ReadMoreLink/readMoreLink.m.css [hunk 6]
```diff
@@ -1,31 +1,18 @@
...
 
-.arrow-right {
-  font-size: 9px;
-}
-
-.text {
-  /* @include font-styles(footer); */
-  margin-right: var(--space1);
+.read-more-link:hover {
+  border: solid 1px #ff5079;
```
Comment (nandap4790) [inline src/components/atoms/ReadMoreLink/readMoreLink.m.css]: here as well

File: src/components/atoms/StoryCardFullWidth/index.js [hunk 7]
```diff
@@ -6,15 +6,15 @@ import { Link } from "@quintype/components";
 import Contributor from "../../atoms/Contributor";
 import ResponsiveImageWithFallback from "../ResponsiveImageWithFallback";
 import { getStoryData, generateImageSources } from "../../utils/utils";
-import Icon from "../Icon";
-import { getMonthDayString } from "../../utils/time-format";
+// import Icon from "../Icon";
+// import { getMonthDayString } from "../../utils/time-format";
 // import { isPremium } from "../../../../isomorphic/data/story";
```
Comment (nandap4790) [inline src/components/atoms/StoryCardFullWidth/index.js]: remove ispremium and icon

File: src/components/atoms/StoryCardFullWidth/index.js [hunk 8]
```diff
@@ -6,15 +6,15 @@ import { Link } from "@quintype/components";
...
 
 import "./storyCardFullWidth.m.css";
 
 export default function StoryCardFullWidth({ story, className = "", cardWithImageZoom = true }) {
   const storyData = getStoryData(story);
-
+  console.log(story);
```
Comment (nandap4790) [inline src/components/atoms/StoryCardFullWidth/index.js]: remove console

File: src/components/atoms/StoryCardFullWidth/index.js [hunk 9]
```diff
@@ -23,13 +23,13 @@ export default function StoryCardFullWidth({ story, className = "", cardWithImag
...
-  const icons = {
-    "photo-album": "gallery",
-    photo: "photostory",
-    video: "video"
-  };
+  // const icons = {
+  //   "photo-album": "gallery",
+  //   photo: "photostory",
+  //   video: "video"
+  // };
```
Comment (nandap4790) [inline src/components/atoms/StoryCardFullWidth/index.js]: remove

File: src/components/atoms/StoryCardFullWidth/index.js [hunk 10]
```diff
@@ -23,13 +23,13 @@ export default function StoryCardFullWidth({ story, className = "", cardWithImag
...
 
-  const iconType = get(icons, [story["story-template"]], null);
+  // const iconType = get(icons, [story["story-template"]], null);
```
Comment (nandap4790) [inline src/components/atoms/StoryCardFullWidth/index.js]: remove

File: src/components/atoms/StoryCardFullWidth/index.js [hunk 11]
```diff
@@ -59,19 +59,21 @@ export default function StoryCardFullWidth({ story, className = "", cardWithImag
               <Icon type={"speaker"} styleName="speaker-icon" />
               {story.engagement.total}
             </span>
-          )}
+          )} */}
```
Comment (nandap4790) [inline src/components/atoms/StoryCardFullWidth/index.js]: remove

File: src/components/atoms/StoryCardFullWidth/storyCardFullWidth.m.css [hunk 12]
```diff
@@ -37,11 +37,11 @@
   width: 200px;
   height: 150px;
   background-image: linear-gradient(225deg, rgba(0, 0, 0, 0) 60%, var(--primary-color));
-}
```
Comment (nandap4790) [inline src/components/atoms/StoryCardFullWidth/storyCardFullWidth.m.css]: remove this

File: src/components/atoms/StoryCardFullWidth/storyCardFullWidth.m.css [hunk 13]
```diff
@@ -50,21 +50,22 @@
...
+    justify-content: center;
   }
 }
 
-.story-byline {
+/* .story-byline {
   display: grid;
   grid-template-areas: "a b";
   color: var(--meta-color);
-}
```
Comment (nandap4790) [inline src/components/atoms/StoryCardFullWidth/storyCardFullWidth.m.css:60]: remove

File: src/components/atoms/StoryCardFullWidth/storyCardFullWidth.m.css [hunk 14]
```diff
@@ -50,21 +50,22 @@
...
+} */
 
-@media (min-width: 768px) {
+/* @media (min-width: 768px) {
   .story-byline {
     margin-top: auto;
   }
-}
-
```
Comment (nandap4790) [inline src/components/atoms/StoryCardFullWidth/storyCardFullWidth.m.css:67]: remove

File: src/components/atoms/StoryCardFullWidth/storyCardFullWidth.m.css [hunk 15]
```diff
@@ -121,7 +122,7 @@
   color: var(--meta-color);
   grid-area: b;
   text-align: right;
-}
+} */
```
Comment (nandap4790) [inline src/components/atoms/StoryCardFullWidth/storyCardFullWidth.m.css]: remove

**Review by nandap4790 — APPROVED** [review]

File: assets/stylesheets/generics/typography.scss (hunk 1 above)
Comment (supreeth-spiger) [reply assets/stylesheets/generics/typography.scss]: done

File: src/components/atoms/BigStoryCard/bigStoryCard.m.css (hunk 2 above)
Comment (supreeth-spiger) [reply src/components/atoms/BigStoryCard/bigStoryCard.m.css]: done

File: src/components/atoms/BigStoryCard/bigStoryCard.m.css (hunk 3 above)
Comment (supreeth-spiger) [reply src/components/atoms/BigStoryCard/bigStoryCard.m.css]: done

File: src/components/atoms/StoryCardFullWidth/index.js (hunk 7 above)
Comment (supreeth-spiger) [reply src/components/atoms/StoryCardFullWidth/index.js]: done

File: src/components/atoms/StoryCardFullWidth/index.js (hunk 8 above)
Comment (supreeth-spiger) [reply src/components/atoms/StoryCardFullWidth/index.js]: done

File: src/components/atoms/StoryCardFullWidth/index.js (hunk 9 above)
Comment (supreeth-spiger) [reply src/components/atoms/StoryCardFullWidth/index.js]: done

File: src/components/atoms/StoryCardFullWidth/index.js (hunk 10 above)
Comment (supreeth-spiger) [reply src/components/atoms/StoryCardFullWidth/index.js]: done

File: src/components/atoms/StoryCardFullWidth/index.js (hunk 11 above)
Comment (supreeth-spiger) [reply src/components/atoms/StoryCardFullWidth/index.js]: done

File: src/components/atoms/StoryCardFullWidth/storyCardFullWidth.m.css (hunk 12 above)
Comment (supreeth-spiger) [reply src/components/atoms/StoryCardFullWidth/storyCardFullWidth.m.css]: done

File: src/components/atoms/StoryCardFullWidth/storyCardFullWidth.m.css (hunk 13 above)
Comment (supreeth-spiger) [reply src/components/atoms/StoryCardFullWidth/storyCardFullWidth.m.css:60]: done

File: src/components/atoms/StoryCardFullWidth/storyCardFullWidth.m.css (hunk 14 above)
Comment (supreeth-spiger) [reply src/components/atoms/StoryCardFullWidth/storyCardFullWidth.m.css:67]: done

File: src/components/atoms/StoryCardFullWidth/storyCardFullWidth.m.css (hunk 15 above)
Comment (supreeth-spiger) [reply src/components/atoms/StoryCardFullWidth/storyCardFullWidth.m.css]: done

**Pull request merged**